serverless-plugin/dashboard.js

```diff
@@ -166,6 +180,38 @@ module.exports = function dashboard (serverless, config, context) {
     return apiWidgets
   }
 
+  /**
+   * Create a set of CloudWatch Dashboard widgets for the Step Function State Machines
+   * CloudFormation resources provided
+   *
+   * @param {object} smResources Object of Step Function State Machine resources by resource name
+   */
+  function createStateMachineWidgets (smResources) {
+    const smWidgets = []
+    for (const res of Object.values(smResources)) {
+      const smName = res.Properties.StateMachineName // TODO: Allow for Ref usage in resource names
```

Comment:
same here, maybe we should create an issue, or refer to #13 (adding step functions in there as well)